capstone/src/main/java/capstone/Extras/Helper.java

```diff
@@ -236,6 +236,26 @@ public static String customer_search() {
     }
   }
 
+  public static Customer customer_search_new() {
```

Comment:
You can replace the old function, unless its still being used, which I think is unlikely since all the function calls that are remaining all use `Database.getCustomer()` in the following lines.